Sources/Basic/Process.swift

```diff
@@ -467,6 +470,18 @@ extension Process {
         return try checkNonZeroExit(arguments: args, environment: environment)
     }
 
+    /// Replace the current process image with a new process image.
+    ///
+    /// - Parameters:
+    ///   - path: The path to the process executable.
+    ///   - arguments: The process arguments.
+    static public func exec(path: String, arguments: String...) throws {
```

Comment:
We should keep this API similar to `Process`'s and extract the path from first argument.